schema/spacewalk/common/tables/endpoint.sql

```diff
@@ -0,0 +1,26 @@
+--
+-- Copyright (c) 2025 SUSE LLC
+--
+-- This software is licensed to you under the GNU General Public License,
+-- version 2 (GPLv2). There is NO WARRANTY for this software, express or
+-- implied, including the implied warranties of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+-- along with this software; if not, see
+-- http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+--
+
+CREATE TABLE access.endpoint (
+    id              BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+    class_method    VARCHAR NOT NULL,
+    endpoint        VARCHAR NOT NULL,
+    http_method     VARCHAR NOT NULL,
+    scope           CHAR(1) NOT NULL
```

Comment:
Scope was moved from namespace to endpoint, when compared to the RFC. Could you elaborate on this option? Should it also be used to namespace, to make it clear when a user want to assign XML-RPC API (and json over HTTP) access only, and which methods should be shown?

Reply:
Then we need to be disciplined to make sure we don't set a namespace starting with API for stuff that only focus on the webUI.
It would also allow to mix webUI features with API (XML-RPC/JSON over HTTP) call, which is a good thing in my opinion.

Reply:
Sure, this is still up for debate. Hopefully we'll get some feedback until the GM and can decide accordingly.